.circleci/config.yml

```diff
@@ -1,24 +1,114 @@
 version: 2.1
 orbs:
-  ruby: circleci/ruby@0.1.2 
+  ruby: circleci/ruby@0.1.2
+
+workflows:
+  version: 2
+  build_and_test:
+    jobs:
+      - test
+      - build
+
+defaults:
+  ruby_image: &ruby_image
+    circleci/ruby:2.5.5-stretch-node
+
+  executor: &executor
+    executor: ruby/default
+
+  restore_cache: &restore_cache
+    keys:
+      - gem-cache-v1-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+      - gem-cache-v1-{{ arch }}-{{ .Branch }}
+      - gem-cache-v1
+
+  install_cmake: &install_cmake
+    name: Install cmake
+    command: |
+      sudo apt-get update
+      sudo apt-get install cmake
+
+  install_dependencies: &install_dependencies
+    name: Install dependencies
+    command: |
+      gem update --system
+      gem i bundler -v $(tail -1 Gemfile.lock | tr -d ' ')
+      bundle install --path vendor/bundle --jobs 4 --retry 3
```

Comment:
You can use new bundler cli syntax instead:
`bundle config set path 'vendor/bundle'`
`bundle install`